README.md

```diff
@@ -1,7 +1,7 @@
 # Cucumber-js
 
 This is the last version in the `cucumber` series. New releases will henceforth
-be made under the `@cucumber/cucumber` package name.
+be made under [the `@cucumber/cucumber` package name](https://www.npmjs.com/package/@cucumber/cucumber).
```

Comment:
I'd say the link should just be for `@cucumber/cucumber` (excluding "the"  and "package name")

Reply:
Yeah that’s neater, updated.